src/python/pants/backend/project_info/tasks/export_version.py

```diff
@@ -13,4 +13,4 @@
 #
 # Note format changes in src/docs/export.md and update the Changelog section.
 #
-DEFAULT_EXPORT_VERSION = '1.0.12'
+DEFAULT_EXPORT_VERSION = '1.0.13'
```

Comment:
Could you add a line to explain 1.0.13? https://github.com/pantsbuild/pants/blob/master/src/docs/export.md#export-format-changes

Reply:
Fixed!